### ISSUE TITLE
Walet Cüzdan Kodları

### ISSUE THREAD

**Issue by Osse07** [IBAX-io/go-ibax#1829]
### Describe the bug

Sürekli hatalı kod uyarısı alıyorum 12 key istiyor ama 15 tane var. Güncellemeden sonra cüzdan bilgilerimde gitti ozaman reedm daki waleti değistirme hakkı verinde oraya aktaralım sinir bozucu 1.5 aydır gorev yapıyorum sinir hastası oldum ya

### Reproduction

123

### System Info

```shell
1235
```

### Logs

```shell
28388ejenen
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by zekxtra** [comment]
aynısı bende de var bağladığım cüzdana erişemiyorum saçmalık. en azından yeni cüzdan eklenmeli

**Comment by Annnaaq** [comment]
Bencede esgi cantaya ulaşamıyorsun

**Comment by Faris733** [comment]
Evet. Aynı sorunlar devam ediyor. Cüzdan yenileme hakkı verin. Eski cüzdan a girilmiyor

**Comment by hsynd58** [comment]
ulaşılıyor

**Comment by Isan9900** [comment]
tanks
